### DISCUSSION TITLE
Timestamp doesn't convert to date in 3.5.2

### DISCUSSION THREAD

**Discussion by siddizi** [chartbrew/chartbrew#231] [Q&A]
Hello guys!
I have a problem with new version. My chart can't convert Timestamp to date. It successfully worked in 3.5.1. After update, I have an issue.
My JSON respone:
![image](https://github.com/chartbrew/chartbrew/assets/40463052/1bc46269-86f0-4002-a3e3-63d7cc79c3f0)

My chart:
![image](https://github.com/chartbrew/chartbrew/assets/40463052/75c13789-f8db-4842-bb19-2c1bfaff928b)

**Comment by razvanilin** [comment]
Hey @siddizi can you check if in the "Configure" tab in your dataset, the "resolved" field is detected as a `date`?
![CleanShot 2024-06-17 at 18 54 36@2x](https://github.com/chartbrew/chartbrew/assets/5342321/824f84c9-03af-443b-8445-6d6d5860ed8c)

**Comment by siddizi** [comment]
Hey @razvanilin !
Yeah. Field "created" is date.
![image](https://github.com/chartbrew/chartbrew/assets/40463052/e86d7c6d-4701-4d6f-b67e-6b718159980e)

**Comment by siddizi** [comment]
I noticed that I only have a problem with my API connection. Everything is fine with SQL.
SQL:
JSON response:
  {
    "ticketid": 3893,
    "dateline": 1689827773,
    "lastactivity": 1690288299,
    "laststaffreplytime": 1689847125,    
  }
![image](https://github.com/chartbrew/chartbrew/assets/40463052/156a8264-f099-4e74-a805-19df7625f3fe)
API:
JSON response:
{
    "summary": "Test1",
    "created": 1717588827228,
    "id": "74-58166",
    "$type": "Issue"
  }
![image](https://github.com/chartbrew/chartbrew/assets/40463052/ce804d39-b162-412e-bce0-ec4de2b7f81d)

**Comment by razvanilin** [comment]
I just noticed that in your API request you have some `resolved` fields that are `null`. Can you try filtering those out to see if it fixes the issue? You can filter at the API level if you have that functionality, or directly in Chartbrew like this:

![CleanShot 2024-06-18 at 09 47 29@2x](https://github.com/chartbrew/chartbrew/assets/5342321/8295a05b-b2ed-4cae-9c96-a76a947f0796)

**Comment by siddizi** [comment]
Unfortunately no. 
![image](https://github.com/chartbrew/chartbrew/assets/40463052/1b97fdb4-9370-4323-bb6d-fb78bd4d081a)
![image](https://github.com/chartbrew/chartbrew/assets/40463052/3ef6e93c-49d4-40e5-8065-a0b8ed44969a)
![image](https://github.com/chartbrew/chartbrew/assets/40463052/7819bb43-f2e6-4c4a-906a-5fa2ed39bf60)

**Reply by razvanilin** [reply]
Thanks for all this info! It seems that it must be something with detecting millisecond timestamps. I'll have a look as soon as I can and update you here on the progress 👌

**Comment by siddizi** [comment]
Thanks a lot for your answers! 
I found workaround. 
I use old file "[determineType.js](https://github.com/chartbrew/chartbrew/blob/021630214e499f68a6df06928ebae1c5300b42d7/server/modules/determineType.js)" in volumes docker-compose. Its working for me. 
`
        volumes:
            - ./files/determineType.js:/code/server/modules/determineType.js
`
